packages/mobile/src/app/Drawers.tsx

```diff
@@ -127,6 +127,8 @@ const commonDrawersMap: { [Modal in Modals]?: ComponentType } = {
   InboxUnavailableModal: InboxUnavailableDrawer,
   LeavingAudiusModal: LeavingAudiusDrawer,
   WaitForDownloadModal: WaitForDownloadDrawer,
+  PublishConfirmation: PublishConfirmationDrawer,
+  EarlyReleaseConfirmation: EarlyReleaseConfirmationDrawer,
```

Comment:
Why aren't all the new ones here? Are the others all rendered inside the forms?

Reply:
Yep! in the navigators